src/libostree/ostree-repo-static-delta-private.h

```diff
@@ -101,8 +102,8 @@ G_BEGIN_DECLS
  * Finally, we have the fallback array, which is the set of objects to
  * fetch individually - the compiler determined it wasn't worth
  * duplicating the space.
- */ 
-#define OSTREE_STATIC_DELTA_SUPERBLOCK_FORMAT "(a{sv}tayay" OSTREE_COMMIT_GVARIANT_STRING "aya" OSTREE_STATIC_DELTA_META_ENTRY_FORMAT "a" OSTREE_STATIC_DELTA_FALLBACK_FORMAT ")"
+ */
+#define OSTREE_STATIC_DELTA_SUPERBLOCK_FORMAT "(a{sv}tayay" OSTREE_COMMIT_GVARIANT_STRING "aya" OSTREE_STATIC_DELTA_META_ENTRY_FORMAT "a" OSTREE_STATIC_DELTA_FALLBACK_FORMAT "a{sv})"
```

Comment:
to discuss

Reply:
This would mean that new ostree versions would read garbage at the end of superblocks on older delta files.

Why not use the `a{sv}` "metadata" dict at the start, which is documented as "The metadata would include things like a version number, as well as extended verification data like a GPG signature."?

Reply:
The superblock is written 'on the fly', so adding an item in "metadata' implies to rewrite it from scratch

Reply:
@em- is right from my perspective.
As far as I remember we planned to construct pseudo-data with all fields included, use it for signing and construct the superblock with this data included.
And we avoid the problem with versions incompatibility in that case.